firmware/src/peripherals.cpp

```diff
@@ -148,9 +148,7 @@ void startPeripherals() {
   // Configure motor PWM timer and pause it
   pwmStart(&PWMD1, &motor_pwm_config);
   PWMD1.tim->CR1 &= ~TIM_CR1_CEN;
-  PWMD1.tim->CR1 = (PWMD1.tim->CR1 & ~TIM_CR1_CMS) 
-                   | TIM_CR1_CMS_0 | TIM_CR1_CMS_1 // (TIM1_CMS = 11) Enable center-aligned PWM
```

Comment:
Is there a reason center-aligned PWM can't be used? I was under the impression that it was better for harmonics, etc